modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/info/collection/provider/test/ManyToManyObjectRelationshipInfoCollectionProviderTest.java

```diff
@@ -159,6 +162,62 @@ public void testManyToManyObjectRelationshipRelatedInfoCollectionProvider()
 		_assertRelatedInfoCollectionProvider(
 			_objectDefinition2, objectDefinition2ObjectEntry,
 			objectDefinition1ObjectEntry1, objectDefinition1ObjectEntry2);
+
+		Map<String, ServiceRegistration<?>> serviceRegistrations =
+			_objectRelationshipLocalService.getServiceRegistrations();
+
+		String serviceRegistrationKey = StringBundler.concat(
+			_objectRelationship.getCompanyId(), StringPool.POUND,
+			_objectRelationship.getObjectRelationshipId());
+
+		ServiceRegistration<?> objectRelationshipServiceRegistration =
+			serviceRegistrations.get(serviceRegistrationKey);
+
+		objectRelationshipServiceRegistration.unregister();
+
+		serviceRegistrations.remove(serviceRegistrationKey);
```

Comment:
Please verify if it is possible to:

- assert that there is no related entry when asserting the _objectDefinition1:
```
_assertRelatedInfoCollectionProvider(
	_objectDefinition1, objectDefinition1ObjectEntry1,
	objectDefinition2ObjectEntry);
_assertRelatedInfoCollectionProvider(
	_objectDefinition1, objectDefinition1ObjectEntry2,
	objectDefinition2ObjectEntry);
```
- assert that there is related entries when asserting the _objectDefinition2:
```
_assertRelatedInfoCollectionProvider(
	_objectDefinition2, objectDefinition2ObjectEntry,
	objectDefinition1ObjectEntry1, objectDefinition1ObjectEntry2);
```